### PR TITLE
.ci/aws: Add stable p3dn testing into CI

### PR DESCRIPTION
**Pull request by a-szegel**
* This changes pins our alinux2 ami to a fixed manually prebuilt AMI.  This is a work around for our image builder problems where we can only build a p3dn ami on a p3dn.
* Adds a 5 minute sleep before launching p3dn's as an attempt to fix the p3dn ICE'ing issues

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rajachan — DISMISSED** [review]

**Comment by a-szegel** [comment]
bot:aws:retest

**Comment by AmedeoSapio** [comment]
bot:aws:retest

**Comment by AmedeoSapio** [comment]
bot:aws:retest

**Comment by AmedeoSapio** [comment]
bot:aws:retest

File: .ci/aws/common.groovy [hunk 1]
```diff
@@ -54,6 +54,12 @@ def wait_for_odcr_capacity(region, instance_count, odcr) {
     sh ". venv/bin/activate; ./PortaFiducia/scripts/wait_for_odcr_capacity.py --region ${region} --odcr-id ${odcr} --required-capacity ${instance_count}"
 }
 
+def p3dn_sleep_5min(instance_type) {
+    if (instance_type == "p3dn.24xlarge") {
+        sh "sleep 300"
```
Comment (bwbarrett) [inline .ci/aws/common.groovy:59]: this is not a fix to the problem.  Why aren't we fixing this with a real retry?

**Review by bwbarrett — CHANGES_REQUESTED** [review]

File: .ci/aws/common.groovy (hunk 1 above)
Comment (a-szegel) [reply .ci/aws/common.groovy]: fixed.

**Comment by a-szegel** [comment]
bot:aws:retest ... checking stability

File: .ci/aws/common.groovy (hunk 1 above)
Comment (bwbarrett) [reply .ci/aws/common.groovy]: I don't see any code where you actually fixed the problem?  It looks like you just took out the hack?

Comment (a-szegel) [reply .ci/aws/common.groovy]: I fixed it in PortaFiducia, I will send you the CR.

**Review by bwbarrett — APPROVED** [review]

**Comment by a-szegel** [comment]
Second run passed, looks more stable.

**Pull request merged**